QUARKUS-4430.md

```diff
@@ -0,0 +1,35 @@
+# QUARKUS-3535 - Add a metric for rejected submissions to the pool in RHBQ 3.2
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-4430
+
+PR: https://github.com/quarkusio/quarkus/pull/39583
+
+Upstream issue: https://github.com/quarkusio/quarkus/issues/39581
+
+Verify that URIs in metrics tags are correctly recorded when they match a client or server pattern,
+especially in cases of redirection (302) and not found (404) responses.
+
+## Scope of the testing
```

Comment:
I'd like you to add test cases for paths with empty path segments; the https://github.com/quarkusio/quarkus/pull/39583/files is comparing paths and we should be sure that it is using normalized paths (which I suppose it does, but let's have tests). e.g. `/moved////{id}`